Integrations/SyslogSender/SyslogSender.py

```
@@ -0,0 +1,310 @@
+
+
+''' Syslog send command '''
+
+
+def syslog_send(manager: SyslogManager, min_severity: int):
+    """
+    Send a message to syslog
+    :param manager: Syslog manager
+    :param min_severity: Minimum severity of incidents to send messages about
+    """
+    message = demisto.args().get('message', '')
+    entry = demisto.args().get('entry')
+    ignore_add_url = demisto.args().get('ignoreAddURL', False)
+    log_level = demisto.args().get('log_level')
+    severity = demisto.args().get('severity')  # From server
+    message_type = demisto.args().get('messageType', '')  # From server
+
+    if severity:
+        try:
+            severity = int(severity)
+        except Exception:
+            severity = None
+
+    if message_type == INCIDENT_OPENED and (severity is not None and severity < min_severity):
+        return
+
+    if not message:
+        message = ''
+
+    message = message.replace('\n', ' ').replace('\r', ' ').replace('`', '')
+    investigation = demisto.investigation()
+    if investigation:
+        investigation_id = investigation.get('id')
+        if entry:
+            message = f'{entry}, {message}'
+        message = f'{investigation_id}, {message}'
+
+    if ignore_add_url and isinstance(ignore_add_url, str):
+        ignore_add_url = bool(strtobool(ignore_add_url))
+    if not ignore_add_url:
+        investigation = demisto.investigation()
+        server_links = demisto.demistoUrls()
+        if investigation:
+            if investigation.get('type') != PLAYGROUND_INVESTIGATION_TYPE:
+                link = server_links.get('warRoom')
+                if link:
+                    if entry:
+                        link += '/' + entry
+                    message += f' {link}'
+            else:
+                link = server_links.get('server', '')
+                if link:
+                    message += f' {link}#/home'
+
+    if not message:
+        return_error('No message received')
```

lets raise an exception instead